### ISSUE TITLE
Is there a simple way to support "|" (pipe) and "&&" in the terminal?

### ISSUE THREAD

**Issue by hengyunabc** [jline/jline3#858]
In the demo, jline-gogo supports them, but it relies on felix gogo, which seems quite complex. Is there a simple way to support "|" and "&&" in the terminal?

**Comment by gnodet** [comment]
> In the demo, jline-gogo supports them, but it relies on felix gogo, which seems quite complex. Is there a simple way to support "|" and "&&" in the terminal?

No, because pipes are not easy to handle.  JLine's gogo demo is really a single class (apart from the other one which belongs to Gogo and has been copied temporarily until gogo is released)...

**Comment by hengyunabc** [comment]
Thank you very much.
